WORKSPACE.in

```diff
@@ -266,3 +266,10 @@ new_local_repository(
     build_file = "third_party/pytorch_gpu.BUILD",
     path = "/usr/local/apollo/libtorch_gpu",
 )
+
+# ffmpeg
+new_local_repository(
+    name = "ffmpeg",
+    build_file = "third_party/ffmpeg.BUILD",
+    path = "/usr/local/apollo/ffmpeg",
+)
```

Comment:
Please hold this part until the image updated, or others will fail to compile.

Reply:
K, thanks